tests/plugins/test_orchestrate_build.py

```diff
@@ -235,12 +238,11 @@ def test_orchestrate_build(tmpdir, caplog, config_kwargs, worker_build_image, lo
         'conf_section': 'worker_x86_64',
         'conf_file': tmpdir + '/osbs.conf',
     }
-    if worker_build_image:
```

Comment:
The test should have `worker_build_image` param removed - and instead verify that `config_kwargs['build_image']` is used as a worker image instead

Reply:
I can remove worke_build_image, but I am keeping it just because we have it in orchestrator_build, even though not used,
test is verifying that value from environ['BUILD'] (orchestrator image) is used as it should,
as config_kwargs['build_image"] is being now overwritten by that

Reply:
I don't see much value in testing deprecated params, as it makes the test a bit more confusion, but I don't mind keeping it